Doc-11534-PM-2.8-Edits-DH  Partner Manager 2.8 edits (#141)
mulesoft/docs-partner-manager: modules/ROOT/Edits/endpoint-as2-send.adoc

= AS2 Send Endpoint Settings

> AS2 send endpoint settings configure the target endpoint in an outbound message flow to which translated B2B/EDI messages are send to partners either directly or via 3rd party connections.

Reviewer: ditto comment to globally replace "/" with either "and" or "or"

Reviewer: ```suggestion
AS2 send endpoint settings configure the target endpoint in an outbound message flow to which translated B2B/EDI messages are sent to partners, either directly or via 3rd party connections.
```

A partner or 3rd party can own AS2 send endpoints. Use a 3rd party-owned AS2 send endpoint for an outbound message flow for partners that use a 3rd party connection.

Create AS2 endpoints by configuring these settings on the partner or 3rd party connection pages:

[%header%autowidth.spread]
|===
|Setting |Description |Required

|Owner
|Partner or 3rd party connection that owns the endpoint.
|Yes

|Usage
|`Send to partners`
|No

|Protocol
|Protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
|Yes

|Partner
|Partner to whom the transaction is sent
|Yes

|Endpoint Name

> |Automatically-generated name that identifies the endpoint. You can modify this value later on.

Reviewer: ```suggestion
|Automatically generated name that identifies the endpoint. You can modify this value later.
```

| Yes

|Description
|User-supplied description of the endpoint's purpose and use.
| No

|Partner AS2 URL

> |Your partner’s or 3rd party connection's AS2 URL to which outbound EDI transmissions are sent.

Reviewer: ```suggestion
|The partner or 3rd party connection AS2 URL to which outbound EDI transmissions are sent.
```

|Yes

|Connection timeout
|Timeout for establishing a connection to the remote service. The default is `30 Seconds`.
|Yes

|Retry interval
|Frequency of reconnection attempts when unable to connect to the remote AS2 service. The default is `30 Seconds`.
|Yes

|Maximum retry attempts
|Maximum number of reconnection attempts to make. If you don't want Partner Manager to retry after an initial connection failure, set this value to `0`.
|Yes

|<Host> AS2 identifier
|Your organization’s AS2 identifier.
|Yes

|Partner AS2 identifier
|Your partner’s or 3rd party connection's AS2 identifier.
|Yes

|Respond to email address

> |Email address to use as the HTTP 'From' header when sending.

Reviewer: Format of "From"?

|Yes

|Message subject
|User-defined field for the MIME Subject header.
|Yes

|Filename pattern
|xref:file-name-pattern.adoc[File name pattern] required by your partner.
|Yes

|Request receipt
a|Whether you want the partner to return an MDN receipt, which confirms the receipt of the message.

|Yes

|<Host> keystore
|The host xref:create-keystore.adoc[keystore] files in your environment that house your organization's public certificate and private key.
|Yes

|Signature algorithm
|xref:as2-endpoints-algorithms.adoc[Algorithm] to use when setting the message integrity check value.
|Yes

|MDN signature algorithm
|Request for a partner to respond with a signed MDN using this xref:as2-endpoints-algorithms.adoc[algorithm] for calculating MIC.
|Yes

|Encryption algorithm
|xref:as2-endpoints-algorithms.adoc[Algorithm] to use when encrypting the message to send.
|Yes

|MDN response

> |The AS2 messages sent from Partner Manager always request the receiver to send a Synchronous MDN receipt back. The default is `Sync`.

Reviewer: ```suggestion
|The AS2 messages sent from Partner Manager always request the receiver to return a synchronous MDN receipt. The default is `Sync`.
```
